### PR TITLE
Task 6

### PR DESCRIPTION
**Pull request by evTyurin**

### CODE REVIEW

File: src/resources/craftsmen.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+id:1,name:bob,surname:smith
```
Comment (Mark005) [inline src/resources/craftsmen.csv]: your data format in CSV file  is incorrect you shouldn't have a key-value format for each field but you have to write data with a header as a first line and data in others 
image

**Review by Mark005 — CHANGES_REQUESTED** [review]

File: src/com/senlainc/warsaw/tyurin/service/OrderService.java [hunk 2]
```diff
@@ -193,15 +194,35 @@ public Order createOrder(String data) {
...
+                    order.setStartDate(LocalDateTime.parse(keyValue.substring(keyValue.indexOf(":") + 1)));
+                } catch (DateTimeParseException e) {
+                    throw new IllegalArgumentException("You enter not a date as submission date");
+                }
             } else if (keyValue.startsWith("completionDate")) {
-                order.setCompletionDate(LocalDateTime.parse(keyValue.substring(keyValue.indexOf(":") + 1)));
+                try {
+                    order.setCompletionDate(LocalDateTime.parse(keyValue.substring(keyValue.indexOf(":") + 1)));
+                } catch (DateTimeParseException e) {
+                    throw new IllegalArgumentException("You enter not a date as submission date");
```
Comment (Mark005) [inline src/com/senlainc/warsaw/tyurin/service/OrderService.java]: you throw the exceptions but where do you handle them??

**Review by mrseneca — APPROVED** [review]
Good job

**Comment by evTyurin** [comment]
> 

Thanks :)

**Pull request merged**